framework/src/multiapps/MultiApp.C

```diff
@@ -554,8 +554,10 @@ MultiApp::backup()
 {
   TIME_SECTION(_backup_timer);
 
+  _console << "Backed up MultiApp ... ";
```

Comment:
One issue is that the caller has this
```
    _console << COLOR_CYAN << "\nBacking Up MultiApps on " << type.name() << COLOR_DEFAULT
             << std::endl;
  ...
    _console << COLOR_CYAN << "Finished Backing Up MultiApps on " << type.name() << "\n"
             << COLOR_DEFAULT << std::endl;
```
If we do not verbose here, the two lines printed by the caller will look worrysome. I personally think adding this is fine since we have control parameters on backup/restore. Seeing this on screen always gives me a peace of mind.

Reply:
But it's definitely debugging information... and not at all useful because you know it happens every time no matter what.

Reply:
We let our users decide? I do not have strong opinions on this although I tend to keep this. It is useful to indicate which multiapp backup/restore has been skipped and show the names of multiapps. Because backup/restore is so significant, not seeing whether or not it is happening is a little worry to me.

Reply:
Yes - you can put this behind a `verbose` option.